cartographer/sensor/voxel_filter.h

```diff
@@ -43,7 +45,14 @@ class VoxelFilter {
   TimedPointCloud Filter(const TimedPointCloud& timed_point_cloud);
 
  private:
-  mapping::HybridGridBase<uint8> voxels_;
+  using KeyType = std::bitset<3 * 8 * sizeof(int)>;
```

Comment:
Same here. Why 8 * sizeof(int) instead of sizeof(uint32)? There are platforms where sizeof(int) != 4.